solr/core/src/java/org/apache/solr/cli/SolrCLI.java

```diff
@@ -444,8 +453,34 @@ private static void printHelp() {
     print("Pass -help or -h after any COMMAND to see command-specific usage information,");
     print("such as:    ./solr start -help or ./solr stop -h");
   }
+
   /**
-   * Get the base URL of a live Solr instance from either the solrUrl command-line option from
+   * Strips off the end of solrUrl any /solr when a legacy solrUrl like http://localhost:8983/solr
+   * is used, and warns those users. In the future we'll have url's with /api as well.
+   *
+   * @param solrUrl with /solr stripped off.
```

Comment:
[Q] Are these Javadocs right?  It looks like the main point of this method is to do the "/solr"-stripping, but this Javadoc makes it sound like the `solrUrl` param is supposed to already be stripped?

Reply:
doh!